v2/internal/controllers/controller_resources.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/Azure/azure-service-operator/v2/pkg/genruntime"
 
 	"github.com/go-logr/logr"
-	corev1 "k8s.io/api/core/v1"
+	coreV1 "k8s.io/api/core/v1"
```

Comment:
minor: Was this fixed in the other commit? Or was it reverted here?

Reply:
Was reverted by rebase; will fix.